<launch>
    <node pkg="hsi_driver" type="cube_data.py" name="profile_node" launch-prefix="valgrind --tool=callgrind --callgrind-out-file=/home/river/test.log">
        <param name="camera_model" value="imec"/>
    </node>
</launch>